docs/users-guide/01-what-is-metabase.md

```diff
@@ -1,6 +1,54 @@
 ## What is Metabase?
-Metabase is an open source business intelligence tool. It lets you ask questions about your data and displays answers in formats that make sense, whether that's a bar graph or a detailed table.  
 
-Your questions can be saved for later, making it easy to come back to them, or you can group questions into great looking dashboards. Metabase also makes it easy to share questions and dashboards with the rest of your team. 
+Metabase is an open source business intelligence tool. It lets you ask questions about your data, and displays answers in formats that make sense, whether that's a bar graph or a detailed table.
+
+Your questions can be saved for later, making it easy to come back to them, or you can group questions into great looking dashboards. Metabase also makes it easy to share questions and dashboards with the rest of your team.
+
+## Finding your way around
+
+So, you've [gotten Metabase up and running](../operations-guide/start.md) and [connected it to your data](../administration-guide/01-managing-databases.md). It's time to give you the lay of the land.
+
+### The home page
+
+![The home page](./images/metabase-homepage.png)
+
+Fresh out of the box, Metabase will show you a few things on the home page:
+* Some [automatic explorations](14-x-rays.md) of your tables that you can look at and save as a dashboard if you like any of them.
+* An area where things you or your teammates create will show up, along with a link to see all the dashboards, questions, and pulses you have.
+* A list of the databases you've connected to Metabase.
+
+![Our data](./images/our-data.png)
+
+Once you've created some [dashboards](07-dashboards.md), any of them that you pin will show up on the homepage for all of your teammates, so that when they log in to Metabase they'll know right where to go.
+
+### Browse your data
+
+![Browse data](./images/browse-data.png)
+
+If you connected your database to Metabase during setup, you'll see it listed at the bottom of the homepage along with the sample dataset that Metabase comes with. Click on a database to see its contents. You can click on a table to see its rows, or you can also click on the bolt icon to x-ray a table and see an automatic exploration of it, or click on the book icon to go to the data reference view for that table to learn more about it.
+
+### Explore your analytics
+
+As long as you're not the first Metabase user on your team, you can browse the dashboards, questions, and collections your teammates have made. [More about exploring](03-basic-exploration.md)
```

Comment:
This first sentence is a bit confusing. I guess its reflecting the fact there might not be stuff on a new instance but not sure we need to call that out explicitly.